### PR TITLE
Use the new organisation schema

### PR DESCRIPTION
**Pull request by rubenarakelyan**
This PR updates the organisation presenter to use the new organisation schema. The new schema contains a lot more information to allow a purely content-store-driven frontend to render organisation pages fully.

It also adds a translation for the "Jobs" corporate information link on organisation homepages to allow it to be translated to Welsh.

Trello: https://trello.com/c/IftdS5pN/25-get-whitehall-to-publish-organisation-data-to-the-content-store

### CODE REVIEW

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 1]
```diff
@@ -26,8 +26,12 @@ def content
...
         document_type: item.class.name.underscore,
         links: {
-          featured_policies: featured_policies_links,
-        },
+          ordered_contacts: contacts_links,
+          ordered_featured_policies: featured_policies_links,
+          ordered_parent_organisations: parent_organisation_links,
+          ordered_child_organisations: child_organisation_links,
+          ordered_successor_organisations: successor_organisation_links,
+        }.compact,
```
Comment (tijmenb) [inline app/presenters/publishing_api/organisation_presenter.rb]: TIL Hash#compact!

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 2]
```diff
@@ -81,14 +109,198 @@ def image
...
+      item.published_corporate_information_pages.map do |cip|
+        {
+          title: cip.title,
+          href: Whitehall.url_maker.public_document_path(cip)
+        }
+      end
+    end
+
+    def featured_links
+      item.featured_links.map do |link|
```
Comment (sihugh) [inline app/presenters/publishing_api/organisation_presenter.rb]: Do these come in any particular order?

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 3]
```diff
@@ -81,14 +109,198 @@ def image
...
+        {
+          title: link.title,
+          href: link.url
+        }
+      end
+    end
+
+    def featured_documents
+      item.feature_list_for_locale(I18n.locale).current.map do |feature|
```
Comment (sihugh) [inline app/presenters/publishing_api/organisation_presenter.rb]: Do these come in any particular order?

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 4]
```diff
@@ -81,14 +109,198 @@ def image
...
+          alt_text: feature.alt_text
+        },
+        summary: offsite_link.summary,
+        public_updated_at: nil, # We don't want a date for offsite links
+        document_type: offsite_link.humanized_link_type
+      }
+    end
+
+    def ministers
+      item.ministerial_roles.map do |role|
```
Comment (sihugh) [inline app/presenters/publishing_api/organisation_presenter.rb]: All the roles are ordered by their presenters, so 👍

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 5]
```diff
@@ -81,14 +109,198 @@ def image
...
+    end
+
+    def special_representatives
+      item.special_representative_roles.map do |role|
+        person(role)
+      end
+    end
+
+    def person(role)
+      person = role.current_person
```
Comment (sihugh) [inline app/presenters/publishing_api/organisation_presenter.rb]: We look forward to migrating the rest of this :-)

File: app/presenters/publishing_api/organisation_presenter.rb (hunk 2 above)
Comment (rubenarakelyan) [reply app/presenters/publishing_api/organisation_presenter.rb]: I'm using `visible_featured_links` now which is what the controller uses without any extra ordering.

File: app/presenters/publishing_api/organisation_presenter.rb (hunk 3 above)
Comment (rubenarakelyan) [reply app/presenters/publishing_api/organisation_presenter.rb]: They are used as-is by the controller at the moment.

File: app/presenters/publishing_api/organisation_presenter.rb (hunk 4 above)
Comment (rubenarakelyan) [reply app/presenters/publishing_api/organisation_presenter.rb]: I've refactored this to reduce duplication and also ensure the ordering is correct.

File: app/presenters/publishing_api/organisation_presenter.rb [hunk 6]
```diff
@@ -81,14 +109,198 @@ def image
...
 
-    def brand
-      brand_colour = item.organisation_brand_colour
-      brand_colour ? brand_colour.class_name : nil
+    def foi_exempt
+      item.foi_exempt
+    end
+
+    def corporate_information_pages
+      item.published_corporate_information_pages.map do |cip|
```
[the remark this answers is not in the document]
Comment (rubenarakelyan) [reply app/presenters/publishing_api/organisation_presenter.rb]: It's actually more complicated than that in the controller/view, so I've replicated the logic they use.

**Pull request merged**